src/sources/aws_kinesis_firehose/mod.rs

```diff
@@ -37,9 +37,9 @@ pub struct AwsKinesisFirehoseConfig {
     ///
     /// AWS Kinesis Firehose can be configured to pass along a user-configurable access key with each request. If
     /// configured, `access_key` should be set to the same value. Otherwise, all requests will be allowed.
-    ///
-    /// This option has been deprecated, the `access_keys` option should be used instead.
-    #[configurable(deprecated)]
+    #[configurable(
+        deprecated = "The `access_key` option has been deprecated, use `access_keys` instead."
```

Comment:
Not a blocker just wondering if we need to specify the field name for all of these since we're already within that section?

Like would this work ?

```suggestion
        deprecated = "This option has been deprecated, use `access_keys` instead."
```

Reply:
Yeah, that's a good call for succinctness.